Runtime/Scripts/FileOpen.cs

```diff
@@ -8,6 +11,12 @@
 #endif
 public class FileOpen : MonoBehaviour
 {
+    [CanBeNull] private Button button;
```

Comment:
is die attribute nodig?

Reply:
Die had ik toegevoegd als hint voor de IDE en een developer om duidelijk te maken dat 'null' een geldige waarde is en invloed heeft op het functioneren; maar ik kan hem ook weghalen :)